packages/utils/test/cacao.spec.ts

```diff
@@ -22,7 +19,7 @@ describe("URI", () => {
     it("should merge recaps", () => {
       const recap1 = {
         att: {
-          "https://notify.walletconnect.com": { "manage/all-apps-notifications": [{}] },
```

Comment:
btw these updates might case tests to fail

Reply:
Curious why ? I checked that notify server also runs under notify.walletconnect.org

Reply:
Ah I just noticed it's because the recap test includes a JWT with the old url. Will revert these back then